src/SelfServe/Example/SelfServeExample.rp.ts

```diff
@@ -0,0 +1,64 @@
+import { get } from "../../Utils/arm/generatedClients/2020-04-01/databaseAccounts";
+import { RefreshResult } from "../SelfServeComponent";
```

Comment:
minimize number if imports.. include userinfo here where userinfo will have rg, subid and acc name